flows/atd-finance-data/finance_data_to_s3.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+
+"""
+Name: ATD Finance Data Flow
+Description: Gets Finance data from a database, places it in an S3 bucket, 
+             then moves it along to Knack and socrata.
+
+Build Deployment yaml file:
+$ prefect deployment build flows/atd-finance-data/finance_data_to_s3.py:main \
+--name "Finance Data Publishing" \
+--cron "13 7 * * *" \
+--pool atd-data-03 -q default \
+-sb github/finance-data-wip \
+-o "deployments/finance_data_to_s3.yaml" \
+--description "Repo: https://github.com/cityofaustin/atd-finance-data, Gets Finance data from a database, places it in an S3 bucket, then moves it along to Knack and socrata."
+
+Then, apply this deployment
+$ prefect deployment apply deployments/finance_data_to_s3.yaml
+"""
+
+import os
+
+import docker
+import prefect
+from datetime import datetime, timedelta
+
+# Prefect
+from prefect import flow, task, get_run_logger
+from prefect.blocks.system import JSON
+
+
+# Docker settings
+docker_env = "production"
+docker_image = "atddocker/atd-finance-data"
+
+
+@task(
+    name="get_env_vars",
+    retries=10,
+    retry_delay_seconds=timedelta(seconds=15).seconds,
```

Comment:
I'm split on whether I should use `timedelta` vs. using an integer for the seconds in the script that I'm working on. Just thinking out loud here!

Also, I am curious about the timeout vs. retry and if there is a reason for the update. I was trying to make the same decision earlier today on the service bot intake script, and I could use your thoughts on this for sure.